db/write_thread.h

```diff
@@ -459,4 +481,13 @@ class WriteThread {
   void CompleteFollower(Writer* w, WriteGroup& write_group);
 };
 
+struct CommitRequest {
+  WriteThread::Writer* writer;
+  uint64_t commit_lsn;
+  std::atomic<bool> applied;
```

Comment:
I suspect this field is no longer useful.

Reply:
The current implementation relies on this, but the variable name is not well expressed in this scenario. 

It is used to prevent when:

Request queue has A and B, A is the first writer, and runs very fast entered commit, but B is just set itself into the request queue. If remove `applied`,  after A pop itself from request queue, it will also commit B directly.

I'll change this when changed to iter wb array.

Reply:
Why? I think `running_nr` can do the job. More specifically, you can set the `running_nr` before adding req to the request queue.

Reply:
indeed :)